lib/node_modules/@stdlib/repl/lib/tokenizer.js

```
@@ -67,11 +67,15 @@ function tokenizer( line, context ) {
 	var i;
 
 	// Parse the given line into tokens & comments...
-	ast = parse( line, {
-		'ecmaVersion': 'latest',
-		'onToken': onToken,
-		'onComment': onComment
-	});
+	try {
+		ast = parse( line, {
+			'ecmaVersion': 'latest',
+			'onToken': onToken,
+			'onComment': onComment
+		});
+	} catch ( error ) { // eslint-disable-line no-unused-vars
+		return tokens;
```

@Snehil-Shah Do you think it is worth using `debug` to log the error here, or do you think it okay to silently swallow the error?

---

Haven't initialized a debugger for this file😬. Although, most errors from here just indicate syntax errors in user's input code (and then we return no tokens), and wouldn't help with REPL's debugging in any way. WDYT?

---

Yes, makes sense. Just wanted to double check.